documentation/_data/sidebars/home_sidebar.yml

```diff
@@ -55,6 +55,9 @@ entries:
       - title: Best Practices
         url: /bestpractices.html
         output: web
+      - title: Large Scale Deployment

```

Comment:
Should we rename it to "Maximum supported scale limit "?

Reply:
I see. Sure.